### ISSUE TITLE
Internal Server Error with tutorial data

### ISSUE THREAD

**Issue by jcpasion** [sdrogers/nplinker#45]
Hello again,

Unrelated to the previous issues I have with Docker, I am going through the Tutorial again, and finished the server loading step. I attempted to connect to `http://localhost:5006/npapp` but I get an 500 internal server error and these output in Docker:

[NPLinker_Error_Log.txt](https://github.com/sdrogers/nplinker/files/6779953/NPLinker_Error_Log.txt)
=

Any help with why I am getting some Value and Attribute errors?  My `.toml` file is using the Duncan dataset + `[webapp]
        tables_metcalf_score = 3.0 `, as per the wiki. Again, thanks for your time!

**Comment by andrewramsay** [comment]
I think this was a simple bug that was triggering the other exceptions you saw as a side-effect, although I'm not sure how it hasn't surfaced before since the responsible part of the code has been there for months!

Anyway I've pushed a small update to the Docker image, if you pull the latest version and try running things again it should work this time. I tested it myself and it all seems to load up OK.
